docs/pages/marketplace/memos.en-US.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Deploy memos
+ogImageTitle: Deploy memos
+ogImageSubtitle: Deploy memos on Zeabur
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Deploy memos
+
+[memos](https://github.com/usememos/memos) A privacy-first, lightweight note-taking service. Easily capture and share your great thoughts.
+
+Here is a live demo of memos hosted on Zeabur: [Demo](https://demo.usememos.com/explore)
+
+This article will show you how to deploy memos on Zeabur in a minute.
+
+## Method 1: Deploy from template(Recommended)
+
+We've built a template for you, what you need is just one click on the button below, and then you will get a running memos service.
```

Comment:
We have built a template for you. All you need to do is click the button below, and you will get a running memos service.